pkg/pgmodel/migrate_test.go

```diff
@@ -598,19 +600,24 @@ func TestSQLJsonLabelArray(t *testing.T) {
 						t.Fatalf("Expected label arrays to be equal: %v != %v", labelArray, labelArrayKV)
 					}
 
-					var jsonres []byte
-					err = db.QueryRow(context.Background(), "SELECT * FROM prom.label_array_to_jsonb($1)", labelArray).Scan(&jsonres)
+					var (
```

Comment:
we should have both `prom.label_array_to_jsonb` and `prom.label_array_to_kv` so we should have tests for both.

Reply:
It's a simple enough wrapper around the array version that I'm not sure it's worth it; it'd require us to re-add the label-set logic for one...

Reply:
it'd require us to re-add the label-set logic for one. isn't that just re-adding lines 630-642 back in?